book/website/communication/citable/citable-linking.md

```diff
@@ -0,0 +1,64 @@
+
+## 2. Keep the Research Objects closed
+
+### Publish the Research Objects under restricted access
+
+Publishing your data under restricted access means that the data is uploaded to the repository, but you're only providing access to individuals that request access or you can provide access through sharing a private link. 
+* You decide with whom to share the private link and whether individuals that request access will obtain access.
+    * You can share the data with reviewers
+* Your metadata (information about the data) is publicly available.
+* You can provide the correct DOI in the paper's **Data Availability Statement** and in the reference list (see {ref}`Citing Data<cm-citable-cite-data>`) 
+* After your article is published: 
+    * Add the DOI of the publication to the dataset 
+    * Change the access permission rights to public (see for example [Zenodo](https://zenodo.org/)).
+
+### Publish the Research Objects under embargo
+
+An embargo means that you have already uploaded the data to the repository, but are not providing access to the data until a certain date. 
+* No one has access to your data until the embargo period has expired (you can define this period), or until you changed the access permissions yourself. 
+    * You cannot share the data with reviewers
+* Your metadata (information about the data) is publicly available.
+* You can provide the correct DOI in the paper's **Data Availability Statement** and in the reference list (see {ref}`Citing Data<cm-citable-cite-data>`) 
+* After your article is published: 
+    * Add the DOI of the publication to the dataset 
+    * Change the access permission rights if the embargo has not yet expired (see for example [Zenodo](https://zenodo.org/)).
+
+### Reserve a DOI and publish later
+
+Several data repositories allow you to reserve a DOI (see for example [Zenodo](https://zenodo.org/), [Figshare](https://figshare.com/))
+Reserving a DOI means you are not required to upload your data yet, but you will need to provide some metadata (title, author/contributors) to complete the DOI reservation. The reserved DOI will be in use after you actually uploaded the data and decide to publish the data. 
+* No one has access to your data or the metadata yet, because the landing page of your dataset is not working yet. 
+    * Your data is not available to reviewers.
+* You can provide the correct DOI in the paper's **Data Availability Statement** and in the reference list (see {ref}`Citing Data<cm-citable-cite-data>`) 
+* After your article is published:
+    * Upload the dataset to your reserved DOI. 
+    * Add the DOI of the publication to the dataset
+    * Publish your dataset
+
+# Check the data DOI in your publication
+
+Whichever route you decide to take, it is important to double check the proof of your article. 
```

Comment:
Suggestion: "double-check" reads better with a hyphen, at least to me (then again, I love hyphenating everything).